src/components/BasicAccordion/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { BasicAccordion } from './BasicAccordion';
```

Comment:
BasicAccordion is a confusing name for a library component, could you please rename it to Accordion only everywhere (including the folder name)